apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -771,10 +772,20 @@ deserialize_transaction(Tx) ->
 %%
 %% One correct solution per blocktime (provided in ms) and 42 graphs per
 %% solution explains the last bit of math.
-target_to_hashrate(Target) ->
+target_to_hashrate(Target, aec_consensus_bitcoin_ng) ->
     case aeminer_pow:scientific_to_integer(Target) of
       0 -> 0;
       _ ->
           Difficulty = aeminer_pow:target_to_difficulty(Target) / (1 bsl 24),
           round(Difficulty * (aec_governance:expected_block_mine_rate() / 1000) * 42)
-    end.
+    end;
+target_to_hashrate(_Target, _Consensus) ->
+    0.
```

Comment:
I believe CT tests use consensus `aec_consensus_common_tests` so this could be a bit unexpected a test if it wants to pretend to mine something. No such test I am aware of, though

Reply:
Good point... This value only makes sense on Mainnet/Testnet so I think we are fine with 0.